backend/processing_image/VERSION

```diff
@@ -1 +1 @@
-8
```

Comment:
Do we want this double bumped?

Reply:
yeah, because I bumped it once already when testing.